src/Components/List/List.tsx

```diff
@@ -168,16 +168,30 @@ const calculateSelectedPosition = (
     return 0
   }
 
-  const itemHeight = 24
+  const heights = {
+    xs: 28,
+    sm: 36,
+    md: 48,
+    lg: 56,
```

Comment:
Where do these numbers come from? Is there a common constant for each size?

Reply:
that's an awesome question. So, I manually did a lookup of the item elements and each of these are sum of height + paddings + margins. I didn't see some common constants for list items.